/**:
  ros__parameters:
    # fiducials_path: kalman_slam/fiducials/erc2024.yaml # Set from the launch file.
    rate: 3.0
    time_offset: 0.3
    odom_frame: map
    robot_frame: base_link
    variance: 5.0
    max_dist: 5.0
    max_num: 1
